cli-tests/tests/flow/pg-operator.spec.ts

```diff
@@ -67,45 +51,17 @@ test.describe('Everest CLI install', async () => {
 
       await out.assertSuccess();
       await out.outContainsNormalizedMany([
-        '✓ Install Operator Lifecycle Manager',
-        '✓ Install Percona OLM Catalog',
-        '✓ Create namespace \'everest-monitoring\'',
-        '✓ Install VictoriaMetrics Operator',
-        '✓ Provision monitoring stack',
-        '✓ Create namespace \'everest-operators\'',
-        '✓ Install operators [pg] in namespace \'everest-operators\'',
-        '✓ Configure RBAC in namespace \'everest-operators\'',
-        '✓ Install Everest Operator',
-        '✓ Install Everest API server',
-      ]);
-    });
-    await page.waitForTimeout(10_000);
-    await verifyClusterResources();
-
-    await test.step('re-run everest install command', async () => {
-      await page.waitForTimeout(60_000);
-      const operator = await cli.exec(`kubectl -n everest-system get po | grep everest|awk {'print $1'}`);
-      await operator.assertSuccess();
-
-      const out = await cli.everestExecSkipWizard(
-        `install -v --operator.mongodb=false --operator.postgresql=true --operator.xtradb-cluster=true --namespaces=everest-operators`,
-      );
-      const restartedOperator = await cli.exec(`kubectl -n everest-system get po | grep everest|awk {'print $1'}`);
-      await restartedOperator.assertSuccess();
-
-      expect(operator.getStdOutLines()[0]).not.toEqual(restartedOperator.getStdOutLines()[0]);
```

Comment:
I removed this test step because Everest operator does not need to restart anymore when the install command is re-run.